data/json/itemgroups/SUS/domestic.json

```
@@ -644,28 +686,38 @@
     "//2": "This group is for things you often find stored on or next to a toilet.",
     "subtype": "collection",
     "entries": [
-      { "distribution": [ { "item": "plunger_toilet", "prob": 90 }, { "item": "plunger_futuristic", "prob": 10 } ] },
+      {
+        "distribution": [ { "item": "plunger_toilet", "prob": 90 }, { "item": "plunger_futuristic", "prob": 10 } ],
+        "prob": 90
+      },
       { "item": "brush_toilet", "prob": 75 },
-      { "item": "toilet_paper", "prob": 80 }
+      { "item": "toilet_paper", "prob": 95, "container-item": "wrapper" },
+      { "item": "bathroom_scale", "prob": 50 }
     ]
   },
   {
     "id": "SUS_bathroom_cabinet",
     "type": "item_group",
     "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
-    "//2": "This group is for a bathroom cabinet.",
+    "//2": "This group is for the cabinet under the sink.",
     "subtype": "collection",
     "entries": [
       { "item": "soap", "count": [ 1, 4 ], "prob": 70 },
       {
         "distribution": [ { "item": "razor_shaving", "count": [ 1, 2 ], "prob": 50 }, { "item": "shavingkit", "prob": 50 } ],
         "prob": 75
       },
+      {
+        "distribution": [ { "item": "bandages", "prob": 60 }, { "item": "medical_gauze", "prob": 20 }, { "item": "1st_aid", "prob": 10 } ],
+        "prob": 40
```

```suggestion
        "prob": 20
```
Generally I think this should be in the medicine cabinet?